index.html

```diff
@@ -1,308 +1,391 @@
+            <h3 class="lang-1">Language</h3>
+          </div>
+          <ul class="work-list">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="language">
+          <div class="language-list">
+            <div class="list-bullet type-3"></div>
+            <h3 class="lang-1">Frameworks</h3>
+          </div>
+          <ul class="work-list">
+            <li>React.JS</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li class="sm-none">Selenium</li>
+          </ul>
+        </div>
+        <div class="language">
+          <div class="language-list">
+            <div class="list-bullet type-2"></div>
+            <h3 class="lang-1">Skills</h3>
+          </div>
+          <ul class="work-list">
+            <li>Database Management</li>
+            <li>Version Control</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Web Development</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <section class="contact">
+      <div class="show-interest">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+      </div>
+      <form
+        action="https://formspree.io/f/mqkonqbq"
+        method="POST"
+        class="contact-form"
+      >
+        <input
+          required
+          type="text"
+          name="fullName"
+          id="fullName"
+          maxlength="30"
+          placeholder="Full name"
+        />
+        <input
+          required
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Email"
+        />
+        <textarea
+          required
+          name="message"
+          id="message"
+          cols="30"
+          rows="5"
+          maxlength="500"
+          placeholder="Write something here..."
+        ></textarea>
+        <button type="submit" class="get-in-touch btn">Get in touch</button>
+      </form>
+    </section>
+    <hr />
+    <footer>
+      <ul class="social-icons">
+        <li>
+          <a href="#">
+            <img src="./assets/Vector (4).png" alt="social icon" class="icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./assets/Group (4).png" alt="linkedIn icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./assets/Vector (5).png" alt="social icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./assets/Vector (6).png" alt="twitter icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./assets/Vector (7).png" alt="medium icon" />
+          </a>
+        </li>
+      </ul>
+    </footer>
+  </body>

```

Comment:
- [ ] I think it would be better for the desktop version to replicate as much as possible the Figma design by:
    - Adjusting the `space` between the **contact section** and the **Footer**, as it is big compared to that of the Figma design 😂 
   ![image](https://user-images.githubusercontent.com/102757126/236503842-0499cb9b-240f-4d05-908a-4a19f07c231d.png)

    - Also aligning to the `left` the form submit **button** for the best user experience and to keep the balance of our design 👍 👍 

    ![image](https://user-images.githubusercontent.com/102757126/236503397-acc15447-d217-4807-96ef-c9109f7acced.png)